manifests/init.pp

```diff
@@ -19,6 +19,9 @@
 # $proxy_password::     Proxy password for authentication
 #                       type:String
 #
+# $pulp_max_speed::     Maximum bandwidth used per download in kb/sec per worker
```

Comment:
Can this match https://github.com/Katello/puppet-pulp/pull/189/files#diff-60ae41fd0a31977447947f59940ee9a4R206 ?